projects/storefrontstyles/scss/components/content/navigation/_page-header.scss

```diff
@@ -0,0 +1,4 @@
+%cx-page-header {
+  position: absolute;
+  margin-top: -10000px;
+}
```

Comment:
In this case, only the element (not the component) needs to be visually hidden. E.g. The hidden H1 in the homepage will look something like this in the code: `<h1 class="cx-visually-hidden">Only available to Screen Reader users</h1>`

Therefore, please create a new class, name it `.cx-visually-hidden` and copy over the properties from Bootstrap `.sr-only` rule as per the screenshot below. Since we will drop Bootstrap at some point we need to create our own class. Imho, this new class should be **global**.

cc. @morganm58 please take note of the new **global** class request, let us know if any objections.

![bootstrap Screen Shot 2021-03-30 at 5 26 44 PM](https://user-images.githubusercontent.com/15113219/113070446-8eadb900-9190-11eb-9bc8-696083a557ee.png)

Reply:
I agree with this approach of Styles vs. negative margin originally proposed in the ticket. 

@Pucek9, I will update the ticket to to reflect this change.

Reply:
> <h1 class="cx-visually-hidden">Only available to Screen Reader users</h1>
@ developpeurweb are there any objections about add this class directly to component instead of h1?

Reply:
@Pucek9 h1 mandatory, please.